examples/README.md

```diff
@@ -0,0 +1,48 @@
+# Merlin Models Example Notebooks
+
+We have created a collection of Jupyter notebooks based on different datasets. These example notebooks demonstrate how to use Merlin Models with TensorFlow.
+
+## Available Example Notebooks
+
+1. **[Getting Started](01-getting-started.ipynb)**: It provides a getting started with Merlin Models. We will train [Facebook's DLRM](https://arxiv.org/pdf/1906.00091.pdf) architecture with only 3 commands.
+
+2. **[Merlin Models and NVTabular: Applying To your own dataset](02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb)**: We will take a look on the `schema` to connect ETL and training step. It will help to apply Merlin Models to your own dataset structures.
```

Comment:
2. **[Applying to your own dataset with Merlin Models and NVTabular](...)**: See how the `schema` object helps connect your feature engineering and training steps. This notebook shows you how to apply Merlin Models to your own dataset.